ApiDocs/serverless.yml

```diff
@@ -1,10 +1,10 @@
 name: talentboard-documentation
 stage: dev
-reskill-docs:
+talentboard-docs:
   component: "@serverless/website"
   inputs:
     code:
       root: ./
       src: ./html-docs
     region: ap-southeast-1
-    bucketName: talentboard-documentaion
\ No newline at end of file
+    bucketName: talentboard-documentation
```

Comment:
By doing this, it will deploy to this S3 bucket and when some one else tries, it will throw an error saying the bucket name is unavailable. We will have to see a better way to pass this

Reply:
Any suggestions to do so?

Reply:
This remark will be addressed in the next PR.